website/gatsby-config.js

```diff
@@ -0,0 +1,83 @@
+const {resolve} = require('path');
+
+// default location for table of contents
+const DOCS = require('../docs/table-of-contents.json');
+
+module.exports = {
+  plugins: [{
+    resolve: `gatsby-theme-ocular`,
+    options: {
+      // Adjusts amount of debug information from gatsby-theme-ocular
+      logLevel: 1,
+
+      DOC_FOLDERS: [`${__dirname}/../docs/`],
+      ROOT_FOLDER: `${__dirname}/../`,
+      DIR_NAME: `${__dirname}`,
+      SOURCE: [`${__dirname}/static`, `${__dirname}/src`],
+
+      PROJECT_TYPE: 'github',
+      PROJECT_NAME: 'H3',
+      PROJECT_ORG: 'uber',
+      PROJECT_URL: 'https://github.com/uber/h3',
+      PROJECT_DESC: 'Hexagonal hierarchical geospatial indexing system.',
+      PATH_PREFIX: '/',
```

Comment:
If the website is still at `https://uber.github.io/h3/` then `PATH_PREFIX` needs to be `/h3`.

I do not believe this is working in the current prod version of gatsby-theme-ocular. A bunch of fixes (as well as breaking changes) are made in 1.2.0-beta.

Reply:
The H3 website is moving to https://h3geo.org so it shouldn't need a prefix anymore. I had difficulty getting the prefixing to work consistently in ocular-gatsby, e.g. I couldn't get it to work for all of images and links.